src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -26,8 +26,6 @@ const QString kRequestPath = QStringLiteral("/ws/2/recording/");
 
 const QByteArray kUserAgentRawHeaderKey = "User-Agent";
 
-mixxx::Duration kDelayDuration = mixxx::Duration::fromNanos(1000000000);
```

Comment:
We need such a define. 
Maybe it can be called. 
`kMinTimeBetweenMbRequests`
Or such.

Reply:
Will it be used, I think this one and `static constexpr int m_DelayDurationMilis = 1000` has the same purpose, right?

Reply:
Delay is the time something is delayed. But The time is the time between two consecutive Requests, not the delay between the responds and a new request. This needs to be part of the name. 
A member variable is visible for every cpp file including the header. We should make sure this is done only when necessary. 
This is why a consexpr in a anonymous namespace is the first choice.

Reply:
That's such a nice explanation. Thank you! 

So I will use the consexpr in the namespace, which is called kMinTimeBetweenMbRequests.

Reply:
Or is it okay to use Duration in the anon namespace?

Reply:
Yes, I am not aware of an issue.